emails/2023-07-14.md

```diff
@@ -0,0 +1,25 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+We had a number of new people subscribe to this list because of our [Golang articles](https://www.dolthub.com/blog/?q=golang). I apologize in advance because your going to get a lot more database version control and open data content than Golang content. We're glad you are here but fair warning.
```

Comment:
```suggestion
We had a number of new people subscribe to this list because of our [Golang articles](https://www.dolthub.com/blog/?q=golang). I apologize in advance because you're going to get a lot more database version control and open data content than Golang content. We're glad you are here but fair warning.
```